content/en/docs/languages/java/configuration.md

```diff
@@ -138,6 +138,8 @@ Properties for disabling the [SDK](../sdk/#opentelemetrysdk):
 returns a minimally configured instance (for example,
 `OpenTelemetrySdk.builder().build()`).
 
+#### Properties: resource
```

Comment:
Bundling in resource configuration options with generate makes it hard to find resource options.